ssl/ssl_sess.c

```diff
@@ -828,6 +828,13 @@ long SSL_SESSION_set_time(SSL_SESSION *s, long t)
     return (t);
 }
 
+int SSL_SESSION_get_protocol_version(const SSL_SESSION *s)
+{
+    if (s == NULL)
+        return (0);
```

Comment:
The `SSL_SESSION_get_time` _et al_ functions do the `NULL` check; I was following that pattern.